src/transformer/fields/adresse/adresse.field.ts

```diff
@@ -35,13 +35,19 @@ const codeInseeIfAny = (codeInsee?: string): { code_insee?: string } => (codeIns
 
 const codePostalFromVoie = (voie: string): string => /\b\d{5}\b/u.exec(voie)?.[0] ?? '';
 
+const getCommuneFromVoie = (voie: string): string =>
+  /\b\d{5}\b\s*(?<commune>\w+)/u.exec(voie)?.groups?.['commune'] ?? ('' as string);
```

Comment:
```suggestion
  /\b\d{5}\b\s*(?<commune>\w+)/u.exec(voie)?.groups?.['commune'] ?? '';
```
`''` est déjà un type string